artifactory/commands/utils/configfile_test.go

```diff
@@ -166,62 +296,66 @@ func TestGradleConfigFileDefaultPatterns(t *testing.T) {
 
 func TestValidateConfigResolver(t *testing.T) {
 	// Create and check empty config
+	tempDirPath := createTempEnv(t)
+	defer os.RemoveAll(tempDirPath)
```

Comment:
Let's assert the error returned from ```os.RemoveAll```. This change is required in a few other places in this file.